Scripts/ci.sh

```diff
@@ -15,7 +15,7 @@ fi
 rm -rf ${PWD}/build
 
 # Run KIF Tests
-env NSUnbufferedIO=YES xcodebuild test -project KIF.xcodeproj -scheme KIF -derivedDataPath=${PWD}/build/KIF -destination "platform=iOS Simulator,${SIMULATOR}" | xcpretty -c
+env NSUnbufferedIO=YES xcodebuild test -project KIF.xcodeproj -scheme KIF -derivedDataPath=${PWD}/build/KIF -destination "platform=iOS Simulator,${SIMULATOR}" CODE_SIGNING_ALLOWED=NO | xcbeautify
```

Comment:
What's the reason for adding `CODE_SIGNING_ALLOWED=NO` here?

Reply:
Because it was failing to build with a you need a signing cert/company chosen.
Tim was trying to fix it here: https://github.com/kif-framework/KIF/pull/1320